apps/developer-portal/src/components/server-configurations/realm-configurations.tsx

```diff
@@ -0,0 +1,303 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AlertInterface, AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import { Field, Forms } from "@wso2is/forms";
+import { Hint, Section } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Divider, Grid } from "semantic-ui-react";
+import { getServerConfigurations, updateServerConfigurations } from "../../api";
+import { SettingsSectionIcons } from "../../configs";
+import { ServerConfigurationsConstants } from "../../constants/server-configurations-constants";
+import { RealmConfigurationsInterface } from "../../models/server-configurations";
+import { URLInputComponent } from "../applications/components";
```

Comment:
Better to move this component to the `components/shared` and use until we move it to `react-components` library.